docs/environment-setup.md

```diff
@@ -107,11 +107,22 @@ _NOTE: This will not work on WSL (Windows Subsystem for Linux). Use the approach
 1. Get [Docker for Desktop For Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac)
 1. Docker for Desktop comes with docker-compose installed.
 
-#### Windows 10 Pro, Enterprise, or Education
+#### Windows 10 Pro, Enterprise, or Education (Hyper-V)
 
 1. Get [Docker for Desktop For Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows)
 1. Docker for Desktop comes with docker-compose installed.
 
+#### Windows 10 Pro, Enterprise, or Education (Insiders / WSL 2 / Docker)
+
+1. If your Windows build is below 1941, join the [insiders program](https://insider.windows.com/en-us/). Then, update your machine to a newer build through Automatic Updates.
```

Comment:
I suggest replacing "If your Windows build is below 1941..." with "If your [Windows 10 build number](https://www.windowscentral.com/how-check-your-windows-10-build) is below [build 18917](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install/)..." (including the hyperlinks, from which I took this information from)